### ISSUE TITLE
Allow users to correct transcriptions

### ISSUE THREAD

**Issue by Michael-F-Bryan** [Michael-F-Bryan/radio-chatter#1]
We'll eventually want to train/fine-tune a model to improve the accuracy for transcribing radio transmissions. To help with that, it'd be really nice if the user could provide corrections through the UI that we can add to the training data.
